.gitignore

```diff
@@ -71,3 +71,4 @@ testem.log
 # System Files
 .DS_Store
 Thumbs.db
+/PhotoPlusBackend/photoplus/src/main/resources/application.yml
```

Comment:
I would not add this file to ignored. There are properties which should be common within system.